packages/talker_flutter/lib/src/ui/theme/talker_screen_theme.dart

```diff
@@ -30,14 +32,23 @@ class TalkerScreenTheme {
     }
     return _defaultColors;
   }
+
+  factory TalkerScreenTheme.fromTheme(ThemeData theme, [LogColors? logColors]) {
+    return TalkerScreenTheme(
+      backgroundColor: theme.colorScheme.background,
+      textColor: theme.colorScheme.onBackground,
+      cardColor: theme.colorScheme.surface,
+    );
+  }
 }
 
-extension MapTalkerFlutterColorsExt on Map<TalkerLogType, Color> {
+extension MapTalkerFlutterColorsExt on LogColors {
   Color getByType(TalkerLogType type) {
     return this[type] ?? Colors.grey;
   }
 }
 
+// 💡 why not merge this into
 const _defaultColors = {
```

Comment:
Sorry I don't got you 🤷‍♂️

Reply:
You can ignore this, forgot to remove it 😅